pkg/core/resources/model/rest/resource_test.go

```diff
@@ -109,28 +110,23 @@ var _ = Describe("Rest Resource", func() {
 				// when
 				rs := rsr.ResourceList
 				// then
-				Expect(rs.Items).To(ConsistOf(
-					&rest.Resource{
-						Meta: rest.ResourceMeta{
-							Type: "TrafficRoute",
-							Mesh: "default",
-							Name: "one",
-						},
-						Spec: &sample_proto.TrafficRoute{
-							Path: "/example",
-						},
-					},
-					&rest.Resource{
-						Meta: rest.ResourceMeta{
-							Type: "TrafficRoute",
-							Mesh: "demo",
-							Name: "two",
-						},
-						Spec: &sample_proto.TrafficRoute{
-							Path: "/another",
-						},
-					}),
-				)
+				Expect(rs.Items).To(HaveLen(2))
+				Expect(rs.Items[0].Meta).To(Equal(rest.ResourceMeta{
```

Comment:
Is the order of items guaranteed?

Reply:
because `ConsistOf` didn't check the order

Reply:
yes, the order is guaranteed